src/components/FormComponents/EditorController/EditorController.styles.ts

```diff
@@ -1,40 +1,49 @@
 import { styled } from '@mui/system';
-import MDEditor from '@uiw/react-md-editor';
 
 import theme from 'styles/theme';
 import { variables } from 'styles/variables';
 
-export const StyledEditor = styled(MDEditor)`
-  &.w-md-editor {
-    border-radius: ${variables.borderRadius.lg2};
-    border: ${variables.borderWidth.md} solid ${variables.palette.surface_variant};
-    background-color: ${variables.palette.surface};
-    color: ${variables.palette.on_surface_variant};
-    font-size: ${variables.font.size.lg};
-    box-shadow: unset;
-  }
+import { MdEditor } from 'components/MarkDownEditor';
```

Comment:
pls add to export to components/index to import MdEditor from components
and remove empty space between styles and components import